course_contents/1_intro/notes/01. Data Types.md

```diff
@@ -0,0 +1,169 @@
+### Variables
+- A variable is a name associated with a value.
+- A variable can be named anything, but it has some rules:
+    1. It cannot start with an integer
+    2. It cannot contain spaces
+    3. It can only contain a symbol, which is the underscore
+
+Examples of variable names:
+- Correct:
+`response`
+`top500`
+`one_apple`
+`first_numbers`
+`_usernames`
+`_last_`
+`current_`
+
+
+- Incorrect:
+`top 500`
+`one apple`
+`1st_numbers`
+`^last_`
+`$current`
+`response#`
```

Comment:
Maybe make these into a single code block? As separate code blocks it will be difficult to design well.

Reply:
Done